scripts/o3de/o3de/manifest.py

```diff
@@ -195,23 +197,25 @@ def load_o3de_manifest(manifest_path: pathlib.Path = None) -> dict:
             return json_data
 
 
-def save_o3de_manifest(json_data: dict, manifest_path: pathlib.Path = None) -> bool:
+def save_o3de_manifest(json_data: dict, manifest_path: pathlib.Path = None) -> None:
     """
-        Save the json dictionary to the supplied manifest file or ~/.o3de/o3de_manifest.json if manifest_path is None
+    Save the json dictionary to the supplied manifest file or ~/.o3de/o3de_manifest.json if None
 
-        :param json_data: dictionary to save in json format at the file path
-        :param manifest_path: optional path to manifest file to save
-        """
+    :param json_data: dictionary to save in json format at the file path
+    :param manifest_path: optional path to manifest file to save
+    """
```

Comment:
The return value is actually used in other [locations](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/register.py#L431)

Reply:
I'll just restore the function to its original state